app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :users
+  after_save :update_posts_counter
+
+  def return_recent_comments
+    comments.order(created_at: :desc).limit(6)
```

Comment:
- [ ] Post model should have a method that returns the 5 most recent comments for a given post, your returning 6 comments, kindly make sure the number of returned comments in 5
`comments.order('created_at Desc').limit(5)`

Reply:
well resolved! ✔️